CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -282,9 +229,67 @@ def add_item(self) -> DB_Response:
     def get_fridge_locations(self):
         pass
 
-class FrigeCheckIn(DB_Item):
-    def __init__(self, db_client: "botocore.client.DynamoDB"):
+
+class FridgeReport(DB_Item):
+
+    REQUIRED_FIELDS = ['fridge_username', 'status', 'notes', 'fridge_percentage']
+    ITEM_TYPES = {
+        'notes': 'S',
+        'fridge_username': 'S',
+        'image_url': 'S',
+        'timestamp': 'N',
+        'status': 'S',
+        'fridge_percentage': 'S'
+    }
+    TABLE_NAME = "fridge_report"
+    VALID_STATUS = set(["working", "needs cleaning", "needs servicing", "not at location"])
+    VALID_FRIDGE_PERCENTAGE = set(["0", "33", "66", "100"])
```

Comment:
1. Any reason why we're using int over float for percentages?
2. If we're rounding off, shouldn't it be `67` instead of `66`?

Reply:
The API contract that frontend and backend team agreed on has ints. I'm not aware of any advantages of using ints over floats for this particular use case, is there one?

Yes it should be 67 if we're rounding off 😅. Good catch

Reply:
Not sure of any obvious advantages, just wanted to know if this was decided for UI purposes (because `33%` vs `33.3%` is a product/design level decision)

Reply:
we won't be displaying "0", "33", "66", "100" in the UI. It's just to give a status to the fullness of the fridge. It could've been [0,1,2,3] too if we wanted to. I didn't have a preference so I just went with what the UI team suggested